reactor-test/build.gradle

```diff
@@ -51,32 +51,30 @@ dependencies {
 	testCompile "org.mockito:mockito-core:$mockitoVersion"
 }
 
-task downloadBaseline {
+task downloadBaseline(type: Download) {
```

Comment:
can you make it part of a separate commit and prepare the PR to be rebase-merged ?